man/ruby.1

```diff
@@ -438,20 +438,22 @@ Show long help message same as
 check of syntax same as
 .Fl c
 .Fl -yydebug
+.Pp
+.El
+.Pp
+.Bl -hang -offset indent
+Or one of the following, which are intended for debugging the interpreter:
 .It Sy yydebug
 compiler debug mode, same as
 .Fl -yydebug
-.Pp
-Only specify this switch if you are going to debug the Ruby interpreter.
 .It Sy parsetree
+Print a textual representation of the Ruby AST for the program
 .It Sy parsetree_with_comment
-AST nodes tree
-.Pp
-Only specify this switch if you are going to debug the Ruby interpreter.
+Print the same tree as parsetree, but with each node annotated with the program code
 .It Sy insns
-disassembled instructions
-.Pp
-Only specify this switch if you are going to debug the Ruby interpreter.
+Print a list of disassembled bytecode instructions.
+.It Sy insns_without_opt
+Print the list of generated bytecode instructions before various optimisations have been applied.
```

Comment:
We should be consistent on whether we use "a list of disassembled bytecode instructions" or "the list of generated bytecode instructions".